modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -720,6 +720,7 @@ Modal.propTypes = {
 				'primary',
 				'secondary',
 				'unstyled',
+				'warning',
```

Comment:
Hey @markocikos I have just added the remaining `displayType` options. Hope to prevent some console warnings from `openModal` calls across the Portal.